pygenesys/utils/db_creator.py

```diff
@@ -562,578 +576,1181 @@ def create_technologies(connector, technology_list):
+                # keep only those vintages that survive to the start of the
+                # simulation
+                years = years[(time_horizon[0] - years) <
+                              tech.tech_lifetime[place]]
+            else:
+                years = time_horizon
+            for emis in emissions_list:
+                db_entry = [(place,
+                             emis.comm_name,
+                             tech.input_comm[place].comm_name,
+                             tech.tech_name,
+                             int(vintage),
+                             tech.output_comm[place].comm_name,
+                             tech.emissions[place][emis],
+                             f"{emis.units}/{tech.output_comm[place].units}",
+                             '') for vintage in years]
+                entries += db_entry
+    cursor.executemany(insert_command, entries)
+    connector.commit()
+    return
+
+
+"""
 def create_():
-CREATE TABLE "MinActivity" (
-	"regions"	text,
-	"periods"	integer,
+CREATE TABLE "tech_exchange" (
 	"tech"	text,
-	"minact"	real,
-	"minact_units"	text,
-	"minact_notes"	text,
-	PRIMARY KEY("regions","periods","tech"),
-	FOREIGN KEY("tech") REFERENCES "technologies"("tech"),
-	FOREIGN KEY("periods") REFERENCES "time_periods"("t_periods")
+	"notes"	TEXT,
+	PRIMARY KEY("tech"),
+	FOREIGN KEY("tech") REFERENCES "technologies"("tech")
 );
 return
 
+
 def create_():
-CREATE TABLE "MaxCapacity" (
-	"regions"	text,
-	"periods"	integer,
+CREATE TABLE "tech_curtailment" (
 	"tech"	text,
-	"maxcap"	real,
-	"maxcap_units"	text,
-	"maxcap_notes"	text,
-	PRIMARY KEY("regions","periods","tech"),
-	FOREIGN KEY("periods") REFERENCES "time_periods"("t_periods"),
+	"notes"	TEXT,
+	PRIMARY KEY("tech"),
 	FOREIGN KEY("tech") REFERENCES "technologies"("tech")
 );
 return
 
+
 def create_():
-CREATE TABLE "MaxActivity" (
-	"regions"	text,
-	"periods"	integer,
+CREATE TABLE "tech_flex" (
 	"tech"	text,
-	"maxact"	real,
-	"maxact_units"	text,
-	"maxact_notes"	text,
-	PRIMARY KEY("regions","periods","tech"),
-	FOREIGN KEY("periods") REFERENCES "time_periods"("t_periods"),
+	"notes"	TEXT,
+	PRIMARY KEY("tech"),
 	FOREIGN KEY("tech") REFERENCES "technologies"("tech")
 );
 return
 
+
 def create_():
-CREATE TABLE "LifetimeTech" (
-	"regions"	text,
+CREATE TABLE "tech_annual" (
 	"tech"	text,
-	"life"	real,
-	"life_notes"	text,
-	PRIMARY KEY("regions","tech"),
+	"notes"	TEXT,
+	PRIMARY KEY("tech"),
 	FOREIGN KEY("tech") REFERENCES "technologies"("tech")
 );
 return
 
+
+
 def create_():
-CREATE TABLE "LifetimeProcess" (
-	"regions"	text,
-	"tech"	text,
-	"vintage"	integer,
-	"life_process"	real,
-	"life_process_notes"	text,
-	PRIMARY KEY("regions","tech","vintage"),
-	FOREIGN KEY("vintage") REFERENCES "time_periods"("t_periods"),
-	FOREIGN KEY("tech") REFERENCES "technologies"("tech")
+CREATE TABLE "groups" (
+	"group_name"	text,
+	"notes"	text,
+	PRIMARY KEY("group_name")
 );
 return
 
 def create_():
```

Comment:
```suggestion
```
It looks like that was hanging around from the previous version.

Reply:
still in a block comment and doesn't do anything